README.md

```diff
@@ -92,137 +54,110 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
-<!-- Features -->
-
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Create animal table**
+- **Populate table with animal data**
+- **Query data**
+- **Update data**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Coming soon...
+> Add a link to your deployed project.
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+- Coming soon....
 
-<!-- GETTING STARTED -->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] Kindly remove this Live Demo section because you don't have any live link for your app now so it would be nice if you delete it now to make your readme clean and professional and with proper information 👍 Once you would have a working live link then you can add this Live Demo section with that link 😊